docs/getting-started.md

```diff
@@ -0,0 +1,168 @@
+
+After some time Unreal Engine will start and you can launch the game.
+
+If you make changes to AirLib you have to run `build.cmd` again.
+
+If you make changes to the plugin code or AirLib, you only have to recompile the plugin.
+This can be done from within the Unreal Editor. go to to `Window` -> `Developer tools` -> `Modules`.
+Search for `AirSim` and click `Recompile`.
+
+#### Launching the game
+
+To run the game, click the big Play button.
+If you want to run it like it would run when packaged, choose 'Run as standalone game'.
+
+## Running the Ros Bridge
+
+The simulator exposes an RPC api that is used by the ros bridge to communicate with the vehicle.
+The ros bridge should preferable run on the same computer as the simulator to ensure low latency and high bandwidth.
+
+> It is theoretically possible to run the ros bridge on a different computer than the simulator.
+  However, this has not been tested recently and it might be broken.
+  You should look into the fsds_ros_bridge.launch file and check the `host_ip` variable.
+
+The ros bridge only works on Ubuntu.
+If you have the simulator running on windows we reccommend Windows Subsystem for Linux.
+This offers a virtual Ubuntu machine within Windows.
+You can read [here how to install it](software-install-instructions.md). 
+While you are at it you might also want to [install Xming](software-install-instructions.md) so you can run rviz and rqt_plot from within WSL.
+
+### Requirements
+
+The ros bridge requires [ROS Melodic to be installed](software-install-instructions.md), as well as the following dependencies:
+```
+sudo apt-get install ros-melodic-tf2-geometry-msgs python-catkin-tools ros-melodic-rqt-multiplot ros-melodic-joy ros-melodic-cv-bridge ros-melodic-image-transport libyaml-cpp-dev libcurl4-openssl-dev
+```
+
+Make sure you have git lfs installed!
+
+### Cloning the repository
+
+Ready? Lets clone the repo into your home directory:
+```
+git clone git@github.com:FS-Online/Formula-Student-Driverless-Simulator.git --recurse-submodules
+```
+
+If you havn't setup your ssh keys, you can clone using https by running the following command:
+```
+git clone https://github.com/FS-Online/Formula-Student-Driverless-Simulator.git --recurse-submodules
+```
+
+**THE REPO HAS TO BE CLONED IN THE HOME DIRECTORY!**. So the repo location should be `$HOME/Formula-Student-Driverless-Simulator`.
+Why you ask? Because we couldn't get relative paths in the C++ code to work so now we have hard-coded some paths to the home directory.
+I know yes it is ugly but it works. If you are bothered by it I would welcome you to open a pr with a fix.
+
+If this folder already exists as a result of any previous step, you can get your settings.json out, delete the folder (or rename it to something like UEProject or UESim) and after cloning the repo place the settings.json file back (this time inside the root of the repository where the default settings.json lives).
```

Comment:
This is hard to follow, could you rewrite it please?

Reply:
Yes!